.travis.yml

```diff
@@ -11,35 +26,46 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=3.2.*
+  matrix: SYMFONY_VERSION=3.4
   global:
     - SYMFONY_DEPRECATIONS_HELPER=48
-    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_VERSION=5.7
+    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
+    - SYMFONY_PHPUNIT_VERSION=5.7
+    - TEST_INSTALLATION=false
 
 matrix:
   include:
     - php: 7.1
+      env: SYMFONY_VERSION=3.4.*
+    - php: 7.1
       env: DEPS=dev SYMFONY_VERSION=3.3.*
-    - php: 5.6
+    - php: 7.1
       env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 7.1
       env: SYMFONY_VERSION=3.1.*
-    - php: hhvm
-      dist: trusty
+    - php: 7.1
+      env: SYMFONY_VERSION=3.2.*
+    - env: TEST_INSTALLATION=true
+
   fast_finish: true
+  allow_failures:
+    - php: 7.1
+      env: SYMFONY_VERSION=3.4.*
+    - env: TEST_INSTALLATION=true
+
 
 before_install:
   - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
   - phpenv config-rm xdebug.ini || true
   - composer self-update
   - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
 
 install: travis_wait composer update --prefer-dist $COMPOSER_FLAGS
 
-before_script: vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh
-
-script: vendor/bin/simple-phpunit
```

Comment:
does make test really run vendor/bin/simple-phpunit or could it be that it runs something else?

Reply:
when debbung into it: yes.